src/vorta/borg/borg_thread.py

```diff
@@ -66,11 +66,7 @@ def __init__(self, cmd, params, parent=None):
         if 'additional_env' in params:
             env = {**env, **params['additional_env']}
 
-        password = params.get('password')
-        if password is not None:
-            env['BORG_PASSPHRASE'] = password
-        else:
-            env['BORG_PASSPHRASE'] = '9999999'  # Set dummy password to avoid prompt.
+        env['BORG_PASSPHRASE'] = params.get('password', '9999999')  # Set dummy password to avoid prompt.
```

Comment:
Thanks for fixing my own bad coding style. 🙈

Reply:
Turns out fixing that broke creating backups, because repositories can be passwordless.